subsys/caf/events/Kconfig

```diff
@@ -53,6 +53,17 @@ config CAF_INIT_LOG_BLE_PEER_CONN_PARAMS_EVENTS
 	depends on LOG
 	default y
 
+config CAF_BLE_SMP_TRANSFER_EVENTS
+	bool "Enable BLE SMP events"
+	help
+	  Enable support for BLE SMP transfer events.
```

Comment:
Do not use `BLE` in docs or Kconfig descriptions. Use `Bluetooth LE`.
Details: https://projecttools.nordicsemi.no/confluence/display/TECHDOC/Terms%2C+abbreviations%2C+and+spelling+rules